src/validations/rules/rules.xsl

```diff
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <xsl:stylesheet
-    exclude-result-prefixes="xs math oscal fv fn sch doc x"
+    exclude-result-prefixes="xs math oscal fv fn sch doc msg x"
     version="3.0"
```

Comment:
How hard would it be to add this to the build, so we can easily view via the Federalist deploy?

Reply:
I mean add rules.xsl as the rendered HTML.

Reply:
We can render HTML (using `rules.xsl` with `rules.xml`+`ssp.sch` as inputs) and produce it as a Federalist deployment artifact. I'd be happy to do so if you can lend a bit of your expertise. All that would be required is a single Saxon-mediated XSL transform.

Reply:
Or did you mean add `rules.xsl `to the UI as an available transform? A transform to HTML or XML or JSON?

Reply:
Sorry for the delay in responding - I meant create the target HTML file. I could coordinate with you on how to do that, and I don't think it needs to be done for this PR.